### PR TITLE
MessageBar modified

### PR DESCRIPTION
**Pull request by BHS-Harish**
**Issue #300  fixed**

**Screenshot**

image

image

### CODE REVIEW

**Comment by BHS-Harish** [comment]
Hey @SamarthKadam 

Please review the request ... Merge it under GSSOC If it's fine

**Assign me the issue #295**

**Comment by SamarthKadam** [comment]
@BHS-Harish decrease gap a bit. seems too much gap. Try to keep it something like what you see in whatsapp web

**Comment by BHS-Harish** [comment]
Hey @SamarthKadam 

For now..?

image

**Comment by SamarthKadam** [comment]
@BHS-Harish please look at the original web app. where you see the timestamp at the right bottom corner. But in your case its coming down. Try to make a clean fork and only make this changes

**Comment by BHS-Harish** [comment]
Hello @SamarthKadam 

I placed the timestamp at bottom right corner

**Comment by SamarthKadam** [comment]
@Saigenix @Spiral-Memory Please review it

File: frontend/src/components/ChatComponents/RecieverMessage.js [hunk 1]
```diff
@@ -42,9 +42,9 @@ export default function RecieverMessage({
             ></Avatar>
           </Tooltip>
         )}
-        <div className="bg-[#FFFFFF]  rounded-tr-xl ml-[1%] font-Roboto rounded-br-xl rounded-bl-xl box-border px-2 py-2 max-[900px]:text-sm flex justify-between">
```
Comment (Spiral-Memory) [inline frontend/src/components/ChatComponents/RecieverMessage.js:45]: Seems like you are also changing the message styles ! Is it necessary ?

File: frontend/src/components/ChatComponents/RecieverMessage.js [hunk 2]
```diff
@@ -42,9 +42,9 @@ export default function RecieverMessage({
...
-        <div>{content}</div>
-          <p className=" text-[11px] pl-2 pt-3 flex items-end font-medium">{`${String(messageTime.getHours()%12 || 12).padStart(2,'0')}:${String(messageTime.getMinutes()).padStart(2,'0')} ${messageTime.getHours()>=12?'pm':'am'}`}</p>
+        <div className="bg-[#FFFFFF] max-w-[100%] relative rounded-tr-lg ml-[1%] font-Roboto rounded-br-lg rounded-bl-lg box-border px-2 pt-2 pb-2 flex flex-col items-end max-[900px]:text-sm flex justify-between">
+          <p className="w-[100%] min-w-[50px] pe-5 pb-2" style={{wordWrap:"break-word"}}>{content}</p>
+          <p className="absolute bottom-[3px] right-2 text-[9px] pl-2 flex items-end font-medium">{`${String(messageTime.getHours() % 12 || 12).padStart(2, '0')}:${String(messageTime.getMinutes()).padStart(2, '0')} ${messageTime.getHours() >= 12 ? 'pm' : 'am'}`}</p>
```
Comment (Spiral-Memory) [inline frontend/src/components/ChatComponents/RecieverMessage.js:47]: Can't we adjust it properly with flex box itself ? Is absolute positioning necessary here ?

File: frontend/src/components/ChatComponents/RecieverMessage.js (hunk 1 above)
Comment (BHS-Harish) [reply frontend/src/components/ChatComponents/RecieverMessage.js:45]: Yes @Spiral-Memory  @SamarthKadam request to make the message similar to whatsapp web So i changed the message style

**Review by Spiral-Memory — APPROVED** [review]

**Comment by BHS-Harish** [comment]
@SamarthKadam 

Can you add gssoc label and level to this PR???

**Pull request merged**